hyp3_gamma/insar/ifm_sentinel.py

```diff
@@ -286,6 +286,14 @@ def make_parameter_file(mydir, parameter_file_name, alooks, rlooks, dem_source):
         f.write('DEM source: %s\n' % dem_source)
         f.write('DEM resolution (m): %s\n' % (res * 2))
         f.write('Unwrapping type: mcf\n')
+        f.write('Reference Point_sar_azlin: %s\n' % coords["row_s"])
+        f.write('Reference Point_sar_rpix: %s\n' % coords["col_s"])
+        f.write('Reference Point_map_row: %s\n' % coords["row_m"])
+        f.write('Reference Point_map_col: %s\n' % coords["col_m"])
+        f.write('Reference Point_map_y: %s\n' % coords["y"])
+        f.write('Reference Point_map_x: %s\n' % coords["x"])
+        f.write('Reference Point_map_lat: %s\n' % coords["lat"])
+        f.write('Reference Point_map_lon: %s\n' % coords["lon"])
```

Comment:
Are `Point_sar_azli` and similar meaningful names? The of the lines in this parameter file use plain language names.